index.js

```diff
@@ -0,0 +1,96 @@
+let firstNumber;
+let display = document.querySelector('.display')
+display.textContent="";
+
+let clear = document.querySelector('#clear')
+let operators = document.querySelectorAll('.operator')
+let operator;
+let results;
+let secondNumber;
+let getElement = document.querySelectorAll('#click')
+let operatorClicked = false;
+
+const sum = ()=>{
+  results = Number(firstNumber + secondNumber); 
+  return firstNumber + secondNumber;
+  
+}
+const substract = ()=>{
+
+  results =  Number(firstNumber - secondNumber);
+  return firstNumber - secondNumber;
+
+}
+const divide = ()=>{
+  results = Number(firstNumber / secondNumber);
+  return firstNumber / secondNumber;
+}
+const multiply = ()=>{
+  results = Number(firstNumber * secondNumber);
+  return firstNumber * secondNumber;
+}
+const operate = () => {
+  if (firstNumber !== undefined && secondNumber !== undefined) {
+    if (operator == '+') {
+      results = sum();
+    } else if (operator == '-') {
+      results = substract();
+    } else if (operator == '/') {
+      results = divide();
+    } else if (operator == '*') {
+      results = multiply();
+    }
+    firstNumber = results;
+    display.textContent = results;
+    operator=undefined
+  }
+}
+
+operators.forEach((ops) => {
+  ops.addEventListener('click', () => {
+    if (firstNumber !== undefined) {
```

Comment:
Your click listeners and operate function have duplicate code. In general it doesn't hurt to better check for undefined / null twice, but usually you only have to do that for interface functions -> functions which are not just used internally.

There is no good concept in JavaScript for private and public functions, but we used to apply a naming convention where we started private functions with an `_` -> e.g. `_operate`.